cwl/default_min.yml

```diff
@@ -0,0 +1,5 @@
+cwl:tool: ./test.cwl
```

Comment:
Can we have slightly more descriptive names, avoiding test or default ?

Reply:
Can you just reject default_min.yml? That would solve the issue.